### ISSUE TITLE
python2Packages.gsd

### ISSUE THREAD

**Issue by freezeboy** [NixOS/nixpkgs#97769]
**Describe the bug**
While working on ZHF, I found the package `pythonPackages.gsd`. its situation is really strange:

- Since 697be75 @jonringer inroduced a specific version freeze to continue tu support python2
- A few days later d1a46d4 @FRidh changed the version to one that doesn't work with python2 (I suspect a bulk update) 

Now python2 is EOL and apparently noone complained about this variation being broken.

2 solutions to fix the hydra failure:

- Revert partially @FRidh commit to get a working gsd package for python2
- Remove this freeze for this package and disable it for python2

I personally prefer the second one, but I can do one or the other, which one is the best for you ?

**Notify maintainers**

cc @costrouc 

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
  - python2Packages.gsd
# a list of nixos modules affected by the problem
module:
```

**Comment by wamserma** [comment]
I'd also vote for dropping the Python2 support.

**Comment by FRidh** [comment]
d1a46d4  is a mistake, and should be reverted

**Comment by FRidh** [comment]
I've reverted it and backported the fix to 20.09.
